src/dowel/csv_output.py

```diff
@@ -72,6 +80,18 @@ def disable_warnings(self):
         """Disable logger warnings for testing."""
         self._disable_warnings = True
 
+    def close(self):
```

Comment:
We really need the file contents to be written out normally as they're provided to the output. Can you think of a way of doing that?